### ISSUE TITLE
[Feature request] linting for plain objects

### ISSUE THREAD

**Issue by brjakab** [francoismassart/eslint-plugin-tailwindcss#229]
**Is your feature request related to a problem? Please describe.**
I want to lint the properties of this object:
```ts
const variantClasses = {
    primary: 'bg-primary text-primary-inverse asdf',
    secondary: 'bg-secondary text-secondary-inverse',
  }
```
The [vscode tailwind intellisense extension](https://github.com/tailwindlabs/tailwindcss-intellisense) can recognise these objects via 
```json
  "tailwindCSS.classAttributes": [
    "class",
    "className",
    "ngClass",
    ".*Classes*"
  ]
```
**Describe the solution you'd like**
The `classRegex` setting could get an array like the classAttributes of the extension mentioned above. Additionally, it should detect plain object names as well.

**Describe alternatives you've considered**
While you can decide to not use such objects or use classNames function for this as well, I think this would be benefitial.

**Additional context**
Please let me know if there is already a solution for this problem.

**Comment by francoismassart** [comment]
Hi @brjakab 

I don't think I'll add sucha fetaure because you can already accomplish what you want using `callees` even if you use a dummy callee like:

```js
const ctl = (classes) => classes
const variantClasses = {
    primary: ctl('bg-primary text-primary-inverse asdf'),
    secondary: ctl('bg-secondary text-secondary-inverse'),
  }
```

Or you can use `cva` which is a great solution for managing variants

https://cva.style/docs/getting-started#your-first-component

**Comment by brjakab** [comment]
Thank you, I will consider these options

**Comment by lukasvice** [comment]
I would also really appreciate if the plugin could detect plain objects, like Tailwind IntelliSense does. That way we wouldn't have to add an empty workaround method to all class declarations, which can be easily forgotten.

It would be nice if 
```
const className = 'flex p-4'
```

would work with the ESLint plugin out of the box, like it does with Tailwind IntelliSense.
